CONTRIBUTING.md

```diff
@@ -97,19 +97,20 @@ to work on.
 
 # 7. Turn coffee and documentation into code and documentation!
```

Comment:
While we're here, I don't think we meant to say documentation twice :slightly_smiling_face: 

```suggestion
# 7. Turn coffee into code and documentation!
```

Reply:
Done in ba160a9.